src/main/java/net/glowstone/command/minecraft/ClearCommand.java

```diff
@@ -215,8 +219,8 @@ private boolean clearAll(CommandSender sender, Player player, Material material,
                 }
             }
         }
-        sender.sendMessage(
-            "Cleared the inventory of " + player.getName() + ", removing " + count + " items");
+        new LocalizedStringImpl("clear.done", resourceBundle).send(
```

Comment:
Could this be changed so the singular form `item` is used when `count == 1`?

Reply:
Done (clear.done.singular).